samu.c

```diff
@@ -163,8 +179,7 @@ main(int argc, char *argv[])
 		buildopts.maxfail = num > 0 ? num : -1;
 		break;
 	case 'l':
-		warn("job scheduling based on load average is not implemented");
-		EARGF(usage());
+		loadflag(EARGF(usage()));
```

Comment:
I'd inline the body of `loadflag` here, to match `-k`.

Reply:
Should it also be inside the body of `parseenvargs()` (or be dropped from SAMUFLAGS)?

Reply:
Since the load average option is effectively the same as the jobs option (in the sense that both have no effect on the build output, only on job scheduling, parallelization) it makes a lot of sense to support -l in the SAMUFLAGS as well.

So yes, I believe it should be read in `parseenvargs()`.

I think there's probably not a good reason to ever drop anything from SAMUFLAGS -- I assume by that you mean silently permit its presence but ignore it.

Reply:
Yep, I just forgot about `SAMUFLAGS`. It's fine as it is.